js/ui/projectManager.js

```diff
@@ -505,12 +515,66 @@ define([
       if (!keepRetained) chrome.storage.local.remove("retainedProject");
       this.render();
     },
-    
+
     getPaths: function() {
       return Object.keys(this.pathMap);
     }
   };
 
+  // Resize sidebar
+  // min / max sidebar width
+  var MIN_WIDTH = 100;
+  var MAX_WIDTH = 500;
+
+  function setupSidebarResize(element, document) {
+    // Sidebar resizing
+    var isResizing = false;
+    // make sure width is not set in element `style`
+    element.style.width = null;
+
+    var startSidebarResize = function (evt) {
+      if (evt.target.hasClass('project-resizer')) {
```

Comment:
Here it is: https://github.com/thomaswilburn/Caret/pull/530/commits/d6d250c899a884b74ec4da4310642a51444dfbd9#diff-4d1f6d0095e22bb74e2b8a36ff770856R30 (sorry it there are a bit too many changes with all the whitespace removed, I put it in a separate commit: https://github.com/thomaswilburn/Caret/pull/530/commits/d6d250c899a884b74ec4da4310642a51444dfbd9)